test/v2/load/load_generator_metrics.go

```diff
@@ -9,8 +9,13 @@ const namespace = "eigenda_test_client"
 
 // loadGeneratorMetrics encapsulates the metrics for the load generator.
 type loadGeneratorMetrics struct {
-	operationsInFlight *prometheus.GaugeVec
-	// TODO (cody-littley) count successes, failures, and timeouts
+	operationsInFlight     *prometheus.GaugeVec
+	writeSuccesses         *prometheus.CounterVec
```

Comment:
Is the common term for this operation writes or dispersals? Or are we okay using them interchangeably.